### PR TITLE
Refactor by using '==' instead of 'is' for comparing booleans and user IDs

### PR DESCRIPTION
**Pull request by clemkoh**
### Description
Needed refactoring for incorrect comparisons.

Fixes part of #201

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Ran unit tests that were already present.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by isabelcosta — APPROVED** [review]
Well done @clemencetbk !

**Comment by isabelcosta** [comment]
@clemencetbk This code addresses part of the issue you mentioned if you wish to leave this as it is, I have no problem merging this. For this, I removed from the PR description the `fixes` keyword, so it won't close the issue #201  

Are you ok with this? I'll merge this once I merge most PRs that have been already approved.

**Comment by clemkoh** [comment]
@isabelcosta Yes, I'm fine with that!

**Pull request merged**
